client/src/components/map.tsx

```diff
@@ -20,17 +21,25 @@ const mapConfig = {
     'carto-light': {
       'type': 'raster',
       'tiles': [
-        'https://a.basemaps.cartocdn.com/light_all/{z}/{x}/{y}@2x.png',
-        'https://b.basemaps.cartocdn.com/light_all/{z}/{x}/{y}@2x.png',
-        'https://c.basemaps.cartocdn.com/light_all/{z}/{x}/{y}@2x.png',
-        'https://d.basemaps.cartocdn.com/light_all/{z}/{x}/{y}@2x.png',
+        'https://a.basemaps.cartocdn.com/light_nolabels/{z}/{x}/{y}.png',
+        'https://b.basemaps.cartocdn.com/light_nolabels/{z}/{x}/{y}.png',
+        'https://c.basemaps.cartocdn.com/light_nolabels/{z}/{x}/{y}.png',
+        'https://d.basemaps.cartocdn.com/light_nolabels/{z}/{x}/{y}.png',
       ],
     },
     'custom': {
-      'projection': 'EPSG:3857',
       'type': 'vector',
       'tiles': [
-        'https://gis.data.census.gov/arcgis/rest/services/Hosted/VT_2019_150_00_PY_D1/VectorTileServer/tile/{z}/{y}/{x}.mvt',
+        'http://localhost:8080/data/tl_2010_bg_with_data/{z}/{x}/{y}.pbf',
```

Comment:
`localhost` url. Maybe add a `todo` to fix it?

Reply:
Good call - in the latest push I use a remote url and add the localhost as a comment